tests/scaletest/scale_test.go

```diff
@@ -189,18 +183,47 @@ func SetupForTesting(t *testing.T) {
 	ingressesUpdated = []string{}
 }
 
-/* Used for Controller and Node reboot */
-func Reboot(t *testing.T, wg *sync.WaitGroup, nodeType string, controllerIP string, username string, password string) {
-	t.Logf("Rebooting %s ... ", nodeType)
-	loginID := username + "@" + controllerIP
-	cmd := exec.Command("sshpass", "-p", password, "ssh", "-t", loginID, " `echo ", password, " |  sudo -S shutdown --reboot 0 && exit `")
-	_, err := cmd.Output()
+func RemoteReboot(user string, addr string, password string, cmd string) (string, error) {
+	_, _ = ssh.ParsePrivateKey([]byte(password))
```

Comment:
we can skip the assignment if the return value are not being used.

Reply:
Done